plugins/meta/portmap/chain.go

```diff
@@ -35,14 +35,19 @@ type chain struct {
 
 // setup idempotently creates the chain. It will not error if the chain exists.
 func (c *chain) setup(ipt *iptables.IPTables) error {
-	// create the chain
+
 	exists, err := chainExists(ipt, c.table, c.name)
 	if err != nil {
 		return err
 	}
+
 	if !exists {
-		if err := ipt.NewChain(c.table, c.name); err != nil {
-			return err
+		err := ipt.NewChain(c.table, c.name)
+		if err != nil {
+			eerr, eok := err.(*iptables.Error)
+			if eok && eerr.ExitStatus() != 1 {
+				return err
+			}
```

Comment:
Can we wrap this up in a function so we don't repeat the lines twice, and also so the function name would be descriptive on what we're checking and why it's ok to ignore the error?

Reply:
Maybe put this function into a new pkg/util/iptables.go file and then we can use it from both plugins easily, since they both need to do the same thing?

Reply:
And it's better to make `1` a constant, seems more meaningful..

Reply:
Sounds good.

There's probably a few other places where iptables usage overlaps between these two plugins as well as `ipmasq` and `bridge`. But maybe for sake of keeping the changeset reasonable, I can do pull just these out to `pkg/util/iptables.go` and then I can come back in a later PR to do that sort of refactoring?

Reply:
Done.